lib/rules/forbid-component-props.js

```diff
@@ -19,6 +19,7 @@ const DEFAULTS = ['className', 'style'];
 
 module.exports = {
   meta: {
+    type: 'suggestion',
```

Comment:
i think this is a "problem" if it's configured to forbid a prop.

Reply:
I was based on the rule category. If the rule category is "Best Practices" then I specified the "suggestion" type.
If we refer the rule to the “problem” type, then we need to change the category. 
Is not it?

Reply:
I'm not sure that's always true. This rule is a "best practice" rule, but violating an explicitly  best practice is a problem.